LanguageFeatures/Primary-constructors/static_processing_A18_t01.dart

```diff
@@ -32,40 +35,16 @@ class C2(String this.v1, int this.v2) {
   int v2;
 }
 
-class C3<T> {
-  String v1;
-  T v2;
-  this(this.v1, T this.v2);
-}
-
-class C4 {
-  String v1;
-  int v2;
-  this(String this.v1, this.v2);
-}
-
-enum E1<T>(T this.v1) {
+enum E<T>(T this.v1) {
```

Comment:
This name probably has to be preserved, or it will clash with line 43.
```suggestion
enum E1<T>(T this.v1) {
```

Reply:
Ups. Thank you for noticing. Fixed.